src/channel.cr

```diff
@@ -1,5 +1,21 @@
 require "fiber"
 
+# Channels enable concurrent communication between fibers.
```

Comment:
```suggestion
# A `Channel` enables concurrent communication between fibers.
```

Singular should be used to describe a singular entity. The plural could be misinterpreted that you'd need multiple channels for concurrent communication.